src/pages/ContactPage.vue

```diff
@@ -63,20 +66,22 @@ h1 {
     }
   }
 
-  &__address {
+  .address-card {
     display: grid;
     grid-template-columns: 32px 300px;
     grid-template-rows: 130px 30px 30px;
     gap: 2.5em;
     width: 100%;
     margin: auto 0;
+    padding: 0;
     padding-left: 4em;
     font-size: 18px;
     font-weight: 400;
     font-family: "Roboto Mono";
     line-height: 32px;
     letter-spacing: 0.015em;
     color: $tuna;
+    border-radius: 0;
   }
```

Comment:
Do we need this? I think most of those styles are already applied by the `AddressCard` component 🤔
If it's just some small modifications we should leave it at only those and remove the duplications 😄

Reply:
Thank you for keeping me on the right track 👼🏽